presentation/src/main/java/team/jsv/icec/ui/main/start/StartActivity.kt

```diff
@@ -0,0 +1,79 @@
+package team.jsv.icec.ui.main.start
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.provider.MediaStore
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import team.jsv.icec.base.BaseActivity
+import team.jsv.icec.ui.camera.CameraActivity
+import team.jsv.icec.ui.main.MainActivity
+import team.jsv.icec.util.PermissionUtil
+import team.jsv.icec.util.requestPermissions
+import team.jsv.presentation.R
+import team.jsv.presentation.databinding.ActivityStartBinding
+
+class StartActivity :
+    BaseActivity<ActivityStartBinding>(R.layout.activity_start) {
+
+    private lateinit var imagePickerLauncher: ActivityResultLauncher<Intent>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        requestPermissions(PermissionUtil.getPermissions())
+        setImagePickerLauncher()
+    }
+
+    override fun onResume() {
+        super.onResume()
+
+        initClickEvent()
+    }
+
+    private fun setImagePickerLauncher() {
+        imagePickerLauncher =
+            registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+                if (result.resultCode == Activity.RESULT_OK) {
+                    val data: Intent? = result.data
+                    val imageUri: Uri? = data?.data
+                    val imagePath: String? = imageUri?.let { uri -> getPathFromUri(this@StartActivity, uri) }
+
+                    startMainActivity(imagePath = imagePath)
+                }
+            }
+    }
+
+    private fun startMainActivity(imagePath: String?) {
+        Intent(this@StartActivity, MainActivity::class.java).apply {
```

Comment:
제가 만들어 둔 확장 함수 사용하면 더 간단하게 사용할 수 있을 것 같습니다!
```suggestion
startActivityWithAnimation<MainActivity>(
    intentBuilder = {
        putExtra("imagePath", imagePath)
    }
)
```